docs/installation.md

```diff
@@ -0,0 +1,382 @@
+# Installaion for production
+
+This page document how to perform a native deployement on a Debian-like environment,
+for a `udata` user with home in`/srv/udata`.
+These steps require the [system dependencies](system-dependencies.md) to be installed.
+
+# User and home dir creation
+
+We want to create a `udata` user which primary group is `udata`,
+member of the `www-data` group and having its home directory in `/srv/udata`:
+
+```shell
+$ useradd -m -d /srv/udata -G www-data udata
+```
+
+You can check the result with:
+
+```shell
+$ id udata
+uid=1001(udata) gid=1001(udata) groups=33(www-data)
+$ ls -l /srv
+total 4
+drwxr-xr-x 9 udata udata 4096 Jun 23 05:50 udata
+```
+
+You can now log into this account to install udata:
+
+```shell
+$ su - udata
+$ pwd
+/srv/udata
+```
+
+## Python and virtual environment
+
+It is recommended to work within a virtualenv to ensure proper dependencies isolation.
+If you're not familiar with that concept, read [Python Virtual Environments - a Primer][].
+
+We create a virtualenv in the udata home directory so it is activated each time
+you log into its account:
+
+```shell
+$ virtualenv --python=python2.7 $HOME
+$ source bin/activate  
+$ pip install Cython  # Enable optimizations on some packages
+$ pip install udata
+```
+You can also install the extensions you want:
+
+```shell
+$ pip install udata-piwik
+```
+
+!!! note
+    We install Cython before all other dependencies because
+    some have an optionnal compilation support for Cython
+    resulting in more performances (mostly XML harvesting).
+
+
+You can now create your configuration file:
+
+```shell
+$ touch udata.cfg
+$ export UDATA_SETTINGS=/srv/udata/udata.cfg
+```
+
+To ease the `udata` client handling, you might want to export this
+environment variable each time you login:
+
+```shell
+$ echo "export UDATA_SETTINGS=/srv/udata/udata.cfg" >> .profile
+```
+
+Then you need to put some configuration parameters in your file.
+See [Adapting settings](adapting-settings.md) for details about the configuration file.
+
+You're all set, you can now use the `udata` command line client to initialize the platform.
+Just answer the questions:
+
+```shell
+$ udata init
+```
+
+## Sample NGinx & uWSGI configuration
+
+You can use whatever stack you want to run udata, NGinx or Apache 2 as reverse proxy, supervisord + GUnicorn or uWSGI...
+
+All you need to remember is that udata requires at least 3 services to run:
+- a web frontend using the `udata.wsgi` WSGI entry point.
+- a worker service using celery
+- a beat/cron service using celery too
+
+We give you an example using [NGinx][] + [uWSGI][] to run these 3 parts,
+expose the frontend on the `open.data.org` domain
```

Comment:
I'm not sure it's standard to use that domain (I immediately checked if you have it 😛), maybe just the classic `example.com`?